manifests/params.pp

```diff
@@ -33,6 +33,35 @@
         ],
         'maxconn' => '8000'
       }
+        $config_file      = '/etc/haproxy/haproxy.cfg'
+      }
+    'FreeBSD': {
+      $package_name     = 'haproxy'
+      $global_options   = {
+        'log'           => [
+          '127.0.0.1 local0',
+          '127.0.0.1 local1 notice',
+        ],
+        'chroot'        => '/usr/share/haproxy',
```

Comment:
I was only thinking that if the port doesn't patch that file (which it doesn't look to) then those are likely just the defaults for haproxy and there may be a more appropriate place on FreeBSD to locate those.  /usr/local/haproxy for example might fit better to the FreeBSD way of doing things.